slave.mk

```diff
@@ -184,6 +184,13 @@ ifeq ($(SONIC_INCLUDE_MUX),y)
 INCLUDE_MUX = y
 endif
 
+ifeq ($(ENABLE_ASAN),y)
+	ifneq ($(CONFIGURED_ARCH),amd64)
```

Comment:
is it known it is not supported on this arch? or it is only because we cannot validate it?

Reply:
yes, the matrix with supported architectures is here:
https://github.com/google/sanitizers/wiki/AddressSanitizer